examples/deployment/docker/db_client/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM golang:1.8
+
+RUN apt-get update && \
+    apt-get install -y mysql-client
+
+ADD . /go/src/github.com/google/trillian
+WORKDIR /go/src/github.com/google/trillian
+
+RUN go get -v ./server/trillian_log_server
+
+ENV DB_USER=test \
+    DB_PASSWORD=zaphod \
+    DB_DATABASE=test
+
+RUN ./examples/deployment/scripts/download-wait-for-it.sh
```

Comment:
Could you mention in the comment that it's used in docker-compose.yaml (or otherwise indicate that it's expected to be run by some monitoring system). This makes it clear that nothing in the container itself uses this.